react/components/EditorContainer/StoreEditor/Store/PWAForm/components/withPWASettings.tsx

```diff
@@ -46,43 +45,31 @@ const options = {
   },
 }
 
-const withPWASettings = (
-  WrappedComponent: React.ComponentType<ManifestData & any>
-) => (props: any) => (
-  <ManifestQuery query={Manifest}>
-    {handleCornerCases(options, ({ data: PWAData, ...restPWAQuery }) => {
-      if (!PWAData.manifest) {
+const withPWASettings = (WrappedComponent: React.ComponentType<PWAData & any>) => (
```

Comment:
I'd avoid using `& any`, as it transforms everything into `any`, which makes type checking kind of useless.